lib/kafo/scenario_manager.rb

```diff
@@ -29,17 +29,18 @@ def available_scenarios
     end
 
     def list_available_scenarios
-      say "Available scenarios"
+      say ::HighLine.color("Available scenarios", :cyan)
       available_scenarios.each do |config_file, content|
         scenario = File.basename(config_file, '.yaml')
-        say "  #{content[:name]} (use: --scenario #{scenario})"
-        say "        " + content[:description] unless (content[:description].nil? || content[:description].empty?)
+        use = (config_file == @previous_scenario ? 'INSTALLED' : "use: --scenario #{scenario}")
+        say ::HighLine.color("  #{content[:name]} ", :yellow)
+        say "(#{use})"
+        say "        " + content[:description] if content[:description]
```

Comment:
this break the listing when description is empty string
![kafo](https://cloud.githubusercontent.com/assets/109773/11956877/d5c1be0c-a8bd-11e5-907c-179d0733449f.png)